lib/index.js

```diff
@@ -345,19 +345,19 @@ setup_brush = function(graph){
           .attr("transform", function(d, i) { return "translate(0,"+ i * 20 +")"; });
 
       legend.append("rect")
-          .attr("x", width + 20)
```

Comment:
Why are we going from width + 20 to an integer (1)

Shouldn't there by a length you pass in?